fs/zbd_zenfs.h

```diff
@@ -152,6 +153,8 @@ class ZonedBlockDevice {
 
   std::shared_ptr<ZenFSMetrics> GetMetrics() { return metrics_; }
 
+  void GetZonesSnapshot(std::vector<ZoneSnapshot> &snapshot);
```

Comment:
Maybe its just me: Renaming this to `GetZoneSnapshots` sounds a little better.

Reply:
Fixed and yes, I also think this is a typo.